### PR TITLE
sarah grubb intake

### PR DESCRIPTION
**Pull request by Henderjay**
This one had some unusual formatting to it. I ended up putting in a ton a small breaks between letter segments. I can explain why if you're confused.

### CODE REVIEW

File: life-and-letters/modernized/07-chapter-6.adoc [hunk 1]
```diff
@@ -2972,25 +3203,28 @@ and our latter end be like theirs.
...
 and on behalf thereof signed by John Davis, Clerk to the meeting this time.
 
-[.asterism]
+[.small-break]
 '''
 
-A Testimony of York quarterly meeting concerning Sarah Grubb,
-late wife of our friend Robert Grubb, of Clonmel, in Ireland.
+[.blurb]
+=== A Testimony of York quarterly meeting concerning Sarah Grubb, late wife of our friend Robert Grubb, of Clonmel, in Ireland.
```
Comment (jaredh159) [inline life-and-letters/modernized/07-chapter-6.adoc]: I wonder about maybe pulling this out into it's own chapter. Thoughts? 🤔

File: life-and-letters/modernized/07-chapter-6.adoc [hunk 2]
```diff
@@ -3068,12 +3302,20 @@ Read, approved, and signed, in, by order,
...
+[.signed-section-signature]
+Mordecai Casson, Clerk to the meeting this time.
 
-[.asterism]
-'''
+[.signed-section-signature]
+Elizabeth Tuke, Clerk this time.
+
+[.blurb]
+=== Some Remarks on Christian Discipline, as it Respects the Education of Youth.
```
Comment (jaredh159) [inline life-and-letters/modernized/07-chapter-6.adoc]: Same with this.

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Looks good.  I think all of the `[.small-break]`s was the right move -- but I also think it's telling us that we're missing a concept or two in our formatting vocabulary and styling. If it becomes clear what that is, we might want to revisit the formatting of this book, but for now, I think this is super good. 👍

The only reason I'm not merging is to discuss the possibility of breaking out those two sections into their own chapters.

**Comment by Henderjay** [comment]
Not sure if you get notifications when I do a new commit on the same branch. I made these two separate chapters at the end.

**Review by jaredh159 — APPROVED** [review]

**Pull request merged**